src/utils/hooks/api/loans/use-get-account-loans-overview.tsx

```diff
@@ -4,14 +4,21 @@ import Big from 'big.js';
 import { useCallback } from 'react';
 
 import { convertMonetaryAmountToValueInUSD } from '@/common/utils/utils';
+import { GOVERNANCE_TOKEN } from '@/config/relay-chains';
 import { BorrowAction, LendAction } from '@/types/loans';
 import { getTokenPrice } from '@/utils/helpers/prices';
 
 import useAccountId from '../../use-account-id';
 import { useGetPrices } from '../use-get-prices';
-import { getTotalEarnedInterestUSDValue, getTotalUSDValueOfPositions } from './get-usd-values';
+import { getPositionsTotalUSDField } from './get-usd-values';
 import { useGetAccountPositions } from './use-get-account-positions';
 
+const getTotalEarnedRewards = (lendPositions: LendPosition[] = [], borrowPositions: BorrowPosition[] = []) =>
```

Comment:
Could be moved to `get-usd-values.ts` with that file being renamed to `utils` or similar :)